packages/db/src/runtime/db-client.ts

```diff
@@ -36,6 +38,44 @@ const remoteResultSchema = z.object({
 });
 
 export function createRemoteDatabaseClient(appToken: string, remoteDbURL: string) {
+	const remoteUrl = new URL(remoteDbURL);
+
+	switch (remoteUrl.protocol) {
+		case 'http:':
+		case 'https:':
+			return createStudioDatabaseClient(appToken, remoteUrl);
+		case 'libsql+http:':
+			remoteUrl.protocol = 'http:';
+			return createRemoteLibSQLClient(appToken, remoteUrl);
+		case 'libsql+https:':
+			remoteUrl.protocol = 'https:';
+			return createRemoteLibSQLClient(appToken, remoteUrl);
+		case 'ws:':
+		case 'wss:':
+		case 'file:':
+		case 'memory:':
+		case 'libsql:':
+			return createRemoteLibSQLClient(appToken, remoteUrl);
+		default:
+			throw new Error(`Unsupported remote DB: ${remoteDbURL}`);
```

Comment:
Why not a `AstroError`

Reply:
It's in an integration, can't use `AstroError`

Reply:
We can import `AstroUserError` though